src/components/atoms/button/button.tsx

```diff
@@ -1,13 +1,17 @@
 import './button.scss';
+import { classNameVariant } from '../../../utils/class-name-variant';
 
 type ButtonProps = {
-  iconName: string,
+  iconName?: 'download' | 'share' | 'citation' | 'follow',
+  variant?: 'action',
```

Comment:
I feel like we could abstract these possible values in a way that can serve the storybook options. But will have a think about this one and perhaps take it on in 10% time.

Reply:
Agreed. They would feel better as an Enum perhaps - each new icon needs a new value.